packages/web-app-files/src/components/FilesList/KeyboardActions.vue

```diff
@@ -34,7 +34,8 @@ export default defineComponent({
   },
 
   computed: {
-    ...mapState('Files', ['latestSelectedId'])
+    ...mapState('Files', ['latestSelectedId']),
+    ...mapGetters('Files', ['files', 'selectedFiles'])
```

Comment:
Why did you add `files` as well? I only see `selectedFiles` being used in the diff.

Reply:
Ah damn, left-over from dev